guides/micronaut-rabbitmq-rpc/bookrecommendation/kotlin/src/main/kotlin/example/micronaut/BookRecommendation.kt

```diff
@@ -0,0 +1,7 @@
+package example.micronaut
+
+import io.micronaut.core.annotation.Creator
+import io.micronaut.core.annotation.Introspected
+
+@Introspected
+class BookRecommendation @Creator constructor(val name: String)
```

Comment:
Is `@Creator` necessary? If you want to include it, we should educate the user about it with a callout.